_docs/ways-to-register-tools.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Different Ways To Register Tools on Dockstore
+permalink: /docs/publisher-tutorials/ways-to-register-tools-on-dockstore/
+---
+
+## Different Ways To Register Tools on Dockstore
+
+There are 3 major ways to register tools on Dockstore
+- The Dockstore website
+- The Dockstore webservice
+- The Write API webservice and client
```

Comment:
Same comments about Write API as above.

Reply:
This should remain and link to the article above on the write API